pkg/cmd/install.go

```diff
@@ -114,6 +114,7 @@ func newCmdInstall(rootCmdOptions *RootCmdOptions) (*cobra.Command, *installCmdO
 	// Maven
 	cmd.Flags().String("maven-local-repository", "", "Path of the local Maven repository")
 	cmd.Flags().StringArray("maven-property", nil, "Add a Maven property")
+	cmd.Flags().StringArray("maven-build-extension", nil, "Add a Maven build extension")
```

Comment:
I'd suggest `maven-extension`. My understanding is that "build" is assumed when it comes to Maven.

Reply:
done